src/Microsoft.AspNet.OData.Shared/Formatter/Serialization/ODataResourceSetSerializer.cs

```diff
@@ -342,6 +396,76 @@ private static Uri GetNestedNextPageLink(ODataSerializerContext writeContext, in
             return null;
         }
 
+        private static Uri GetNestedNextPageLink(ODataSerializerContext writeContext, int pageSize, object obj = null)
```

Comment:
> = null [](start = 110, length = 7)

This is a new private method. Why make this parameter optional, rather than passing in null?